sound/soc/sof/intel/hda-dai.c

```diff
@@ -525,6 +525,9 @@ int sdw_hda_dai_hw_params(struct snd_pcm_substream *substream,
 		return ret;
 	}
 
+	if (sdev->dspless_mode_selected)
+		goto skip_tlv;
```

Comment:
or just `return 0;` without a new label?

Reply:
I was trying to follow [skip_tlv:](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/intel/hda-dai.c#L426) in `non_hda_dai_hw_params_data`. But I am fine to just return 0. Will follow the majority.

Reply:
The initial skip_tlv isn't really useful, I am not sure why I used a goto. I would use return 0 everywhere. gotos make sense when error control gets out of hand and when the indentation level is starting to get in the way of readability. For a single level branch it should be avoided.

Reply:
Updated